templates/portal/home.html.twig

```diff
@@ -34,6 +36,10 @@
             return true;
         }
 
+        function xl(string) {
```

Comment:
isn't this essentially a function that does not do what it is supposed to do :)

would just use xlj via php/twig instead rather than doing this

Reply:
Nope I did on purpose. I need to have the xl() js so I can use my js utilities alerts etc. especially when I start doing notifications. I planned to add the functionality just not now....

Reply:
Is it necessary/used in current codebase (if fix below use of the xl)? If not needed, then would just wait until have the functionality. Otherwise the 6.1.0 release will have xl calls that can not be translated.

Reply:
answered my own question, looks like you are using utility.js function below, so guessing will break if don't have the xl(). We will need to prioritize getting xl() js translation mechanism in portal to work for 6.1.0.

Reply:
@bradymiller For an additional 2 atta boys I'll go ahead and add it now.:)
However, you should know me now by now. I actually do come back to stuff..

Reply:
no prob if want to come back (plan release 6.1.0 in 2 weeks :) )
here are the atta boys though :)

![2DV](https://user-images.githubusercontent.com/278968/155012265-65902c96-763e-4d80-8683-438f6521e3e2.gif)

![118382341-53d9d600-b5c2-11eb-8b7e-220bbab8468a](https://user-images.githubusercontent.com/278968/155012299-faba5dbe-0499-433c-981e-9f99ea991baa.gif)

Reply:
Inflation and all. Next week I'll need 3 for any basic work!:)